examples/MQ/pixelSimSplit/src/devices/FairMQChunkMerger.cxx

```diff
@@ -0,0 +1,191 @@
+/********************************************************************************
+ *    Copyright (C) 2014 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH    *
+ *                                                                              *
+ *              This software is distributed under the terms of the             *
+ *              GNU Lesser General Public Licence (LGPL) version 3,             *
+ *                  copied verbatim in the file "LICENSE"                       *
+ ********************************************************************************/
+/**
+ * FairMQChunkMerger.cxx
+ *
+ * @since 2016-03-21
+ * @author R. Karabowicz
+ */
+
+#include <boost/thread.hpp>
```

Comment:
I think these are unused and can go away. Even if they are used, can be replaced with std::thread.